docs/getting-started.md

```diff
@@ -1,28 +1,31 @@
 # Getting Started
 > **Note**:  this project is heavily WIP and until it reaches v1.0 the API is subject to change in breaking ways without notice.
 
-## 1 Introduction
+## 0 Introduction
```

Comment:
I like the `0` cause there are no actual steps involved to get it running, just reading through the theory. Then the work starts at step `1`

Reply:
I'm okay keeping it with 0 for now, because i think this whole doc can be designed a lot better. Having the whole entire docs in the readme makes a long doc. maybe we sepearte section 0 into another file next week